CHANGELOG_PENDING.md

```diff
@@ -8,6 +8,7 @@
 
 - \#1848 Use fee cut instead of fee share for user facing language in the CLI (@kyriediculous)
 - \#1854 Allow to pass region in the custom s3 storage URL (@darkdarkdragon)
+- \#1873 Increase TicketParams expiration to 10 blocks (@kyriediculous)
```

Comment:
I think this should be under "Orchestrator" since it is an orchestrator implementation detail.